### ISSUE TITLE
Reloading VR Scene changes settings icon position(Only in IOS)

### ISSUE THREAD

**Issue by mnitish** [googlevr/gvr-unity-sdk#784]
I have two scenes in my game: 
1) Menu scene
2) Vr game scene

In Menu scene I have forced the screen orientation to portrait.

Found using:
 * Google VR SDK version: 1.110
 * Unity version: 2017.2
 * Phone manufacturer, model, and O/S version: iPhone 5, iOS 10
 * Viewer manufacturer & model: ___

Steps to reproduce the issue:
 1. Start the game go from Menu scene to VR game scene. App works fine.
 2. Go back to menu scene and go to vR game scene again, settings icon(gear) is shifted to upper middle to screen.
3. Go back to menu scene and go to VR scene again, now settings icon is shifted to upper left i.e. in place of back arrow. 

Workarounds:
If I don't force the screen orientation to portrait in Menu scene it works fine.

Additional comments:
-- This is not an issue in case of android Devices, only occurring in IoS devices

**Comment by rusmaxham** [comment]
We're taking a look.

**Comment by sanjayc77** [comment]
The UIViewController hosting the VR scene has to support landscape right orientation only:
```
- (UIInterfaceOrientationMask)supportedInterfaceOrientations {
  return UIInterfaceOrientationMaskLandscapeRight;
}
````
If you are using the same view controller for menu and vr scene, separating them may be the way to go.

**Comment by mnitish** [comment]
Solved it!!!

When going back to menu scene from VR scene we were forcing screen orientation to portrait in Menu screen Start() function, this was creating the issue. Now what I have done is, when going back to menu screen in Start function, instead to forcing it to go Portrait strait, we have oriented it to landscape first and then in a coroutine we have oriented it to Portrait mode. This resolved the mentioned issue. 

@rusmaxham : I hope this can give you guys  some pointers to resolve this issue in SDK.
Feel free to contact me if any other information is required!
